src/components/modals/AutoLogoutModal.tsx

```diff
@@ -39,52 +57,103 @@ export const AutoLogoutModal = (props: Props) => {
     bridge.resolve(undefined)
   })
 
-  const numberPickerOptions = [<Picker.Item key="disable" label={lstrings.string_disable} value={0} color={textColor} />]
-  for (let i = 1; i < 60; i++) {
-    const text = String(i)
-    numberPickerOptions.push(<Picker.Item key={text} label={text} value={i} color={textColor} />)
-  }
+  // In Android, the Picker will be hidden with the text overlay.
+  // Determine correct contrasting text color for the native Android Picker
+  // menu. The selected option will show the normal theme color in a text
+  // overlay, but the Picker's color itself must be set to a color that
+  // contrasts with the Android OS theme.
+  React.useEffect(() => {
+    if (isAndroid) {
+      const { UIManagerModule } = NativeModules
+      const uiMode = UIManagerModule && UIManagerModule.getConstants().UI_MODE_NIGHT_YES
```

Comment:
This query isn't async, so we don't need the `useState` / `useEffect` pair. You could instead use `androidNativeTextColor = React.useMemo(..., [theme, isAndroid])`, which would still query the OS only once, but also save a render.